nixos/modules/services/cluster/kubernetes/kubelet.nix

```diff
@@ -79,6 +79,12 @@ in
       type = str;
     };
 
+    extraSANs = mkOption {
+      description = "Extra x509 Subject Alternative Names to be added to the kubelet tls cert.";
+      default = [];
+      type = listOf str;
```

Comment:
can you add an `example`, please? like I did in https://github.com/NixOS/nixpkgs/pull/74094